inspirehep/modules/theme/jinja2filters.py

```diff
@@ -684,21 +684,19 @@ def format_date(datetext):
 @blueprint.app_template_filter()
 def find_collection_from_url(url):
     """Returns the collection based on the URL."""
-    if 'conferences' in url:
+    if 'cc=conferences' in url or '/conferences' in url:
```

Comment:
This current code will cover all use cases from now until we add a new collection. It’s not the most elegant, but sometimes it doesn’t have to be.
All we’d be doing is coming up with a more convoluted way of writing what we have here.

> On 27 May 2016, at 16:35, Samuele Kaplun <notifications@github.com> wrote:
> 
> In inspirehep/modules/theme/jinja2filters.py <https://github.com/inspirehep/inspire-next/pull/1151#discussion_r64913997>:
> 
> > @@ -684,21 +684,19 @@ def format_date(datetext):
> >  @blueprint.app_template_filter()
> >  def find_collection_from_url(url):
> >      """Returns the collection based on the URL."""
> > -    if 'conferences' in url:
> > +    if 'cc=conferences' in url or '/conferences' in url:
> Would be great if this was actually done by properly parsing the URL query and path. In this way we would avoid unexpected behaviors.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/inspirehep/inspire-next/pull/1151/files/519460768c6ee6cfe7076d9b8bec0d917e5ee39f#r64913997>, or mute the thread <https://github.com/notifications/unsubscribe/AARPHFj4z-LCbSAFaqa_mhFkiT2V_Gxbks5qFwEmgaJpZM4InToo>.
>